src/source/PeerConnection/Rtp.h

```diff
@@ -52,6 +64,8 @@ typedef struct {
     UINT32 jitterBufferSsrc;
     PJitterBuffer pJitterBuffer;
 
+    PRollingBufferConfig pRollingBufferConfig;
```

Comment:
My main concern is that this might contain junk values since the user is supposed to create this struct in application and pass to SDK if it's not initialized. Would suggest either:
- Creating a new versioned struct (e.g. advancedSettings) with a cmake flag toggle to pass this rollingBufferConfig and any additional future fields.
- Creating a new constructor for peerconnection (e.g. peerconnectionWithRollingBufferConfig or createPeerConnectionWithAdvancedSettings) to allow passing this new input variable

Reply:
Discard the junk values since struct this is constructed internally by the SDK.